actions/v3/upload_sessions.go

```diff
@@ -43,71 +46,69 @@ type UploadSessionCreateResV3 struct {
 	UploadBeta  bucketRequest `json:"uploadBeta"`
 }
 
+var NumChunksLimit = -1 //unlimited
+
+func init() {
+	if v, err := strconv.Atoi(os.Getenv("NUM_CHUNKS_LIMIT")); err == nil {
+		NumChunksLimit = v
+	}
+}
+
 // Update uploads a chunk associated with an upload session.
 func (usr *UploadSessionResourceV3) Update(c buffalo.Context) error {
-	start := PrometheusWrapper.TimeNow()
-	defer PrometheusWrapper.HistogramSeconds(PrometheusWrapper.HistogramUploadSessionResourceUpdate, start)
+	return c.Render(202, actions_utils.Render.JSON(map[string]bool{"success": true}))
+}
```

Comment:
Is this intended to be just a stand-in for now?

Reply:
Sorry, it is still in progress.

Reply:
Sorry, I did not notice the "WIP"

Reply:
Lol, updated the title